tests/unit/library.cairo

```diff
@@ -154,6 +159,8 @@ namespace it {
 
         %{
             stop_prank = start_prank(ids.caller) 
+            mock_call(ids.old_to_owner, "onERC3525Received", [ids.IERC3525_RECEIVER_ID])
+            mock_call(ids.old_to_owner, "supportsInterface", [1])
```

Comment:
to be consistent with the other test, do we want to change `[1]` to `[ids.TRUE]`  ?